opentelemetry-api-shaded-for-instrumenting/build.gradle.kts

```diff
@@ -38,6 +38,13 @@ val v1_31Deps by configurations.creating {
   exclude("io.opentelemetry", "opentelemetry-bom")
   exclude("io.opentelemetry", "opentelemetry-bom-alpha")
 }
+val v1_32Deps by configurations.creating {
```

Comment:
this part does not come from https://github.com/open-telemetry/opentelemetry-java-instrumentation/commit/c4d14f9ef6b3cdb2540a726d4093b22ba8da55f7 which is mentioned in the pr title

Reply:
Ugh. Ok. 😞